lib/improver/tests/nowcasting/lightning/test_NowcastLightning.py

```diff
@@ -708,8 +708,8 @@ def test_ice_large_with_fc(self):
         self.fg_cube.coord('forecast_period').points = [3600]  # 1 hour
         fg_cube_next = self.fg_cube.copy()
         time_pt, = self.fg_cube.coord('time').points
-        fg_cube_next.coord('time').points = [time_pt + 7200]  # 2 hours
-        fg_cube_next.coord('forecast_period').points = [10800]  # 3 hours
+        fg_cube_next.coord('time').points = [time_pt + 5400]  # 1.5 hours
+        fg_cube_next.coord('forecast_period').points = [9000]  # 2.5 hours
```

Comment:
Should we still have one forecast period of more than 2.5 hours?  Or is this covered elsewhere?

Reply:
Agreed. I've added this back in.